### PR TITLE
feat(uat): codebuild config

### PR DESCRIPTION
**Pull request by esmirnov-klika-tech**
**Issue #, if available:**
https://klika-tech.atlassian.net/browse/GGMQ-211

**Description of changes:**
- added buildspec for Codebuild
- added github job

**Why is this change necessary:**
- to automate uat testing

**Any additional information or context required to review the change:**
It is required to configure the AWS credentials: https://github.com/aws-actions/configure-aws-credentials

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .github/workflows/otf.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          role-to-assume: ${{ vars.AWS_ROLE_TO_ASSUME }}
+          role-session-name: clientDeviceAuthCI
+          aws-region: ${{ vars.AWS_REGION }}
+      - name: Run UAT on linux
+        uses: aws-actions/aws-codebuild-run-build@v1
+        with:
+          project-name: ${{ vars.CODE_BUILD_PROJECT_LINUX }}
+          buildspec-override: uat/codebuild/uat_linux_buildspec.yaml
```
Comment (bgklika) [inline .github/workflows/otf.yml]: Please add \n after last line in file

File: uat/codebuild/uat_linux_buildspec.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+reports:
+  uat-reports:
+    files:
+      - "TEST-greengrass-results.xml"
+    file-format: "JUNITXML"
+
+cache:
+  paths:
+    - '/root/.m2/**/*'
```
Comment (bgklika) [inline uat/codebuild/uat_linux_buildspec.yaml]: Please add \n after last line in file

**Review by bgklika — DISMISSED** [review]

File: .github/workflows/otf.yml (hunk 1 above)
Comment (esmirnov-klika-tech) [reply .github/workflows/otf.yml]: fixed 49a65321497ac954bf2c0147688d8c1fc42df424

File: uat/codebuild/uat_linux_buildspec.yaml (hunk 2 above)
Comment (esmirnov-klika-tech) [reply uat/codebuild/uat_linux_buildspec.yaml]: fixed 49a65321497ac954bf2c0147688d8c1fc42df424

**Review by auarbekov — APPROVED** [review]

**Pull request closed without merging**